content/en/blog/2025/devex-survey/index.md

```diff
@@ -0,0 +1,148 @@
+and we have a
+[repository](https://github.com/open-telemetry/sig-developer-experience) of our
+own on GitHub.
+
+## Key Takeaways
+
+The survey responses highlight the need for improved documentation, more
+in-depth examples and more substantial guidance on debugging, configuring and
+deploying OpenTelemetry in Production. Below are the main areas of concern:
+
+- Documentation:
+  - Documentation is difficult to navigate, requiring users to switch between
+    the OpenTelemetry website and GitHub repositories. Many users report having
+    to piece together information from various sources.
+  - Lack of examples of production usage for both the SDKs and Collector.
+- Local debugging:
+  - It's difficult to know why exporting isn't working, and if the SDK is
+    dropping data. Better tooling for local debugging would help with
+    troubleshooting.
+- SDK configuration:
+  - Overall complexity (both configuration and concepts)
+  - SDK configuration is complex, and many users have pointed out that
+    troubleshooting the exporter is the most painful part.
+- Collector operations:
+  - Debugging guidance on the Collector configuration and its components is
+    needed. OTTL was mentioned a couple of times for being difficult to
+    troubleshoot.
+
+## Detailed Insights
+
+### General Information about Respondents
+
+The survey responses came from users, of whom the majority use OpenTelemetry in
+production (83%) and do not work for a vendor (77.4%). 82% consider their
+organization to be at an intermediate or advanced phase of their journey with
+observability. The platforms used leaned towards Kubernetes (68.2%) and AWS
+(34.6%), with bare metal still making up a respectable percentage (12%).
+Programming language usage was split between the usual suspects of Go, Java,
+JavaScript, Python and .NET, but had a wide spread and even languages without
+official API/SDK implementations, as shown in the second graph below. While
+those working on backends or operations were the majority of respondents, mobile
+and browser were represented as well, see the third graph:
+
+![Forms response chart. Question title: Where in your Observability journey is
+your organization?. Number of responses: 217 responses.](image1.png)
+
+![](image2.png)
+
+![Forms response chart. Question title: What kind of service do you regularly
+work on? (select all that apply). Number of responses: 217
+responses.](image3.png)
+
+### Documentation
+
+Results showed that, by far, people were getting their day one information when
+getting started with OpenTelemetry from the website and GitHub repositories for
+languages, 75.1% and 43.8%, while 25.1% were starting from their vendor’s
+documentation. However, unclear documentation took the top spot in the three
+questions related to getting started with manual instrumentation, auto
+instrumentation and setting up the SDK. In the collector's case, it took second
```

Comment:
We capitalize Collector: https://opentelemetry.io/docs/contributing/style-guide/

Reply:
Thanks, done.